_elements/form-controls.md

```diff
@@ -30,7 +30,7 @@ lead: Intro text on what is included in this section and how to use it. No more
 
   <div class="usa-input-error">
     <legend>
-      <span for="input-error">Text Input Error</span>
+      <span class="usa-input-label">Text Input Error</span>
```

Comment:
@msecret I thought it was only if it was on a label, like the others. Do you think it is?